extensions/molecule/service_cluster_mock/converge.yml

```
@@ -157,32 +183,44 @@
 
     - name: Assert update changed (http persistent)
       ansible.builtin.assert:
-        that: update_result_http_persistent is changed
-        fail_msg: "Update (http persistent) should report changed."
-...
+        that:
+          - update_result_http_persistent is changed
+          - update_result_http_persistent.service_cluster.upstream_hostname == "192.168.1.102"
+          - update_result_http_persistent.service_cluster.outlier_detection_split_external_local_origin_errors == false
+          - update_result_http_persistent.service_cluster.outlier_detection_consecutive_local_origin_failure == 50
+        fail_msg: "Update (http persistent) should report changed with new outlier options. update_result_http_persistent={{ update_result_http_persistent }}"
```

YAMLlint (1.37.1)

[error] missing document end "..."

(document-end)